.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "explorer.compactFolders": false
+}
```

Comment:
@bechurch should this be here?

Reply:
Nope. Weridly enough its marked as an exception to our no `.vscode` files rule in our gitignore.

Ill remove both the file and that exception